docs/user/cassandra/geoserver.rst

```diff
@@ -1,43 +1,28 @@
-.. _create_cassandra_ds_geoserver:
-
 Using the Cassandra Data Store in GeoServer
 ===========================================
 
 .. note::
 
-    See :doc:`/user/geoserver` for general information for working with GeoMesa in Geoserver.
-
-On the "Add Store" page, select "Cassandra (GeoMesa)" as shown below.
-
-.. image:: /user/_static/img/CassandraNewDataSource.png
-
-In the following page, enter the name and description of the data store, the contact point
-(HOST:PORT), the keyspace and the catalog. Click "Save", and GeoServer will search Cassandra for any
-GeoMesa-managed feature types in that keyspace.
+    For general information on working with GeoMesa GeoServer plugins,
+    see :doc:`/user/geoserver`.
 
-.. image:: /user/_static/img/CassandraDSParams.png
-
-Note that when you create the data store, you'll be asked
-to specify a work space name. Remember this name because you'll need to use it when querying the
-data store in GeoServer.
-
-.. note::
+From the main GeoServer page, create a new store by either clicking
+"Add stores" in the middle of the **Welcome** page, or anywhere in the
+interface by clicking "Data > Stores" in the left-hand menu and then
+clicking "Add new Store".
 
-    Note that the "Layer Preview" within GeoServer generally will not work with a Cassandra
-    data store, because the Cassandra data store expects queries to have both a ``bbox`` and
-    date ``between`` components, but the layer previews do not provide these.
+If you have properly installed the GeoMesa Cassandra GeoServer plugin as described
+in :ref:`install_cassandra_geoserver`, "Cassandra (GeoMesa)" should be included in the list
+under **Vector Data Sources**. If you do not see this, check that you unpacked the
+plugin JARs into in the right directory and restart GeoServer.
 
-Once you have a Cassandra layer set up in GeoServer you can try some queries. One way to test queries against the
-GeoServer layer is to submit HTTP requests for the WMS and WFS services. For example, assuming
-you have ingested the ``example.csv`` dataset as described above and set it up as a layer in GeoServer, this request should return
-a PNG image with a single dot::
+On the "Add Store" page, select "Cassandra (GeoMesa)". The Cassandra data store requires three parameters:
```

Comment:
Can we list the other parameters that are inherited from ``GeoMesaDataStoreFactory``, even if they're not required to run the C* data store? Since they shared among multiple data stores, maybe they can go in ``/user/datastores/*`` (and maybe this should be a separate PR).